mac

```diff
@@ -22,12 +22,12 @@ mysql_password="root"
 mysqldump="/Applications/MAMP/Library/bin/mysqldump"
 dropbox_folder="/Volumes/Cloud/Dropbox/"
 php="/Applications/MAMP/bin/php/php5.5.26/bin/php"
-php_cli="/usr/local/bin/php"
+php_cli=$(which php)
 phpmyadmin="http://localhost/phpMyAdmin/"
-github_email_account="guarinogabriel@gmail.com"
-github_username="guarinogabriel"
+github_email_account=$(git config -l | grep user.email=.* | cut -d "=" -f2)
```

Comment:
@andypham This command is not working:
`$(git config -l | grep user.email=.* | cut -d "=" -f2)`

It should be modified to:
`$(git config -l | grep user.email= | cut -d "=" -f2)`